### ISSUE TITLE
{CLI}:unable to get local issuer certificate.

### ISSUE THREAD

**Issue by satushe** [aws/aws-cdk#22234]
### Describe the bug

I have installed the AWS CDK and AWS CLI on my machine. where I am getting the below error while executing the script 
Error: unable to get local issuer certificate

Please find the below details :
O/S : Windows 10

AWS CLI version : 
aws-cli/1.20.49 Python/3.6.0 Windows/10 botocore/1.21.49
AWS CDK version :
2.43.1

I am posting this. as I see the similar issue for one of the user reported on the thread : #8733 . but I could not get the resolution over there. hence creating new one.
 

### Expected Behavior

Should able to create a stack in AWS.

### Current Behavior

Getting the below error : unable to get local issuer certificate

### Reproduction Steps

1. Tried with the hello-cdk application https://cdkworkshop.com/20-typescript.html on the local. 
2. while executing the cdk bootstrap command getting the Error : unable to get local issuer certificate

### CDK CLI Version

2.43.1

### Node.js Version

v16.4.2

### OS

window 10

### Language

Typescript

**Comment by peterwoodworth** [comment]
Hmm I'm afraid we'll need more information to help you out here. The reply in the other thread mentioned this error comes from VSCode. Are you using VSCode?

Additionally, it seems the other issue may have been related to being on a proxy. Are you on a proxy?

Also, could you post the full error logs when you attempt to bootstrap, thanks!

**Comment by satushe** [comment]
@peterwoodworth : Thanks for the reply. Yes I am using VSCode as well as CMD Prompt of Windows 10. and also I am using proxy too.
I am getting  the error in both VSCode and CMD Prompt. 

![Error_CDK_CLI](https://user-images.githubusercontent.com/32052028/192434092-600317bf-81bd-4309-9f98-f62b38793a11.png)

**Comment by ArazHeydarov** [comment]
@rix0rrr, in my case, I'm sure that NodeJs is picking up my proxy certificate since I've added that by exporting the `NODE_EXTRA_CA_CERTS` env variable. And I've tested by using a simple HTTP request script that was not working without that env variable. I thought maybe `cdk deploy` creates and runs some kind of script that doesn't run my `./zshrc` exported variables. But I don't think that's the case. Any other ideas?
